lib/jwt/jwk/rsa.rb

```diff
@@ -83,15 +91,9 @@ def jwk_attributes(jwk_data, *attributes)
           end
         end
 
-        def rsa_pkey(rsa_parameters)
-          raise JWT::JWKError, 'Key format is invalid for RSA' unless rsa_parameters[:n] && rsa_parameters[:e]
-
-          create_rsa_key(rsa_parameters)
-        end
-
         if ::JWT.openssl_3?
           ASN1_SEQUENCE = %i[n e d p q dp dq qi].freeze
-          def create_rsa_key(rsa_parameters)
+          def rsa_pkey(rsa_parameters)
```

Comment:
Why not `create_*`?

Reply:
There was an intermediate function `rsa_pkey` which was only conditionally throwing an error and then calling the other one.
At some point it made sense to move the error checking to `import` instead (this is also consistent with **EC** and **oct**), so `rsa_pkey` simply became an alias for `create_rsa_key`. Having an alias for a private function did not make sense IMO, so I removed the indirection. Why `rsa_pkey` instead of `create_rsa_key`? No reason, except that there is a function `ec_pkey` already, so this seemed more consistent to a reader.

Reply:
I have now reverted the name change and instead renamed `ec_pkey` to `create_ec_key` as well. This is probably the most readable solution :)